ddtrace/internal/utils/http.py

```diff
@@ -209,3 +210,15 @@ def w3c_encode_tag(args):
     tag_val = pattern.sub(replacement, tag_val)
     # replace = with ~ if it wasn't already replaced by the regex
     return tag_val.replace("=", "~")
+
+
+def get_64_lowest_order_bits_as_int(large_int):
```

Comment:
these don't really make sense as `http` utilities, maybe make more sense in `span` or in `tracer` since they pertain to the implementation of trace ids specifically

Reply:
I added these here because these utils were mainly used in propagation. But that changed so now it makes no sense. I’ll make this change